### ISSUE TITLE
app crashing

### ISSUE THREAD

**Issue by briannyeko** [voisine/breadwallet#375]
Hi! the app is crashing when trying to open it on iOS(9.3.1)

**Comment by voisine** [comment]
Hi Brian, can you turn on settings->privacy->diagnostics & usage->automatically send if it's not on already? This will allow us to get the crash report

**Comment by briannyeko** [comment]
hi Aaron! Fixed the problem, its was the iphone itself not the app, my bad!

**Comment by voisine** [comment]
Well now I'm curious, was the phone damaged or defective or something? Also was it jailbroken?

**Comment by dnivi3** [comment]
I'm having the same issue. Jailbroken iOS 9.0.2 5S. Can provide crash log if necessary. I had no issues before this update.

**Comment by voisine** [comment]
@dnivi3 that would be great. Also if you send your iTunes account email address to support@breadwallet.com, we can give you a testfilght invite to try out a new build that I think might fix the issue.

**Comment by dnivi3** [comment]
Done, sent the email a few hours ago. Contains crash log and Apple ID email address for TestFlight.

**Comment by dnivi3** [comment]
@voisine: any update on this? I've submitted crash logs and my Apple ID email through the email you provided.

**Comment by voisine** [comment]
Yes, we will have an update shortly that I think will resolve it. Should be in testflight later today.

**Comment by Nicodemus75** [comment]
I am having the same problem. iphone 6Plus, ios 8.4 Jailbroken

I will email crash report and Apple ID as well

**Comment by voisine** [comment]
can you try removing/uninstalling winterboard and see if that's the culprit? If so, we will need to submit a crash report to the winterboard author.

**Comment by soofuzz** [comment]
Hey, 

Running 9.0.2 on 6+
The crashes recently started on 0.6.2. I'll go ahead and send the full report to the email above as well as an iTunes account for test flight build to help crush bugs.

**Comment by voisine** [comment]
@soofuzz can you check if winterboard is the culprit? I think we will need to submit a crash report to the winterboard author.

**Comment by soofuzz** [comment]
@voisine winter board doesn't seem to be the culprit. it's not installed on my device. I went ahead and sent the crash logs and added an edit to my OP on how i fixed it. totally forgot you can directly downgrade the app on the app store with that tweak. 

Based @Unlimapps Inc for this nice tool.

**Comment by voisine** [comment]
@soofuzz you said "You'll get a notification about allowing notifications to come through."

Can you explain what that is? Is this a notification from breadwallet, or from "app admin"? If you say no does it still crash?

**Comment by soofuzz** [comment]
@voisine its the app asking permission to allow push notifications. app admin doesn't plug into other apps. its only a plugin for the app store to request certain software identifiers (app versions) for download. or just block updates for certain apps you add to a list.

**Comment by jmacwhyte** [comment]
@soofuzz You said disabling Cydia lets the app start just fine, so it must be a substrate tweak that's causing the crash. Not much we can do about that, I don't think. It might be a good idea to turn off all tweaks and enable them one at a time to see which one is causing the problem.

**Comment by soofuzz** [comment]
@jmacwhyte seems like that. but the app is crashing again. going to try to determine what the issue is. Just held up in travels and hopefully sometime this weekend or late next week i can sit down and track the issue.

**Comment by Cabalist** [comment]
My app is crashing in iOS 11.0.2 & 11.0.3  across two devices.  It only happens after I recover my wallet.  I have emailed support but not heard back...

**Comment by jmacwhyte** [comment]
@Cabalist Ryan...? You should just email me directly :D
